platform/fabric/core/generic/config/config.go

```diff
@@ -198,14 +198,16 @@ func (c *Config) GetPath(key string) string {
 	return c.configService.GetPath("fabric." + c.prefix + key)
 }
 
-func (c *Config) MSPCacheSize(defaultValue int) int {
+const DefaultCacheSize = 3
```

Comment:
Please, move this const to the top of the file.
Shall we call it DefaultMSPCacheSize?
Please, add also a comment describing the constant.

Reply:
OK ! making the change